packages/manager/apps/dedicated/client/app/ip/byoip/byoip.constants.js

```diff
@@ -0,0 +1,13 @@
+export const AS_OPTIONS = ['ovh_cloud', 'own'];
+export const CONFIG_NAME = {
+  CAMPUS: 'campus',
+  IPRIR: 'ipRir',
+};
+
+export const IPV4_BLOCK_PATTERN = /^(?:(?:25[0-5]|(2[0-4]|1\d|[1-9]|)\d)(\.(?!$)|$)){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)((\/([0-9]|[1-2][0-9]|3[0-2]))?)$/;
```

Comment:
One regex more easy for pattern IP_V4:
```
export const IPV4_BLOCK_PATTERN = /^((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)(\.(?!$)|$)){4}$/;

```

Reply:
This does work for` IPV4` but not for the `block` pattern of `1.1.1.1/12`